spec/rails_helper.rb

```diff
@@ -73,3 +73,7 @@
     with.library :rails
   end
 end
```

Comment:
Hi @firstpj, @Kaiserabbas, @Xghayor,

Great effort on your project! 👏 However, it seems there are some missing features in your test suite. To meet the project requirements, please make sure your tests cover the following aspects:

#### User Index Page:
- [ ] Tests for displaying the username of all other users.
- [ ] Tests for displaying the profile picture for each user.
- [ ] Tests for displaying the number of posts each user has written.
- [ ] Tests for redirecting to the user's show page when clicking on a user.

#### User Show Page:
- [ ] Tests for displaying the user's profile picture.
- [ ] Tests for displaying the user's username.
- [ ] Tests for displaying the number of posts the user has written.
- [ ] Tests for displaying the user's bio.
- [ ] Tests for displaying the user's first 3 posts.
- [ ] Tests for displaying a button to view all of a user's posts.
- [ ] Tests for redirecting to the post's show page when clicking on a user's post.
- [ ] Tests for redirecting to the user's post's index page when clicking on "see all posts."

#### User Post Index Page:
- [ ] Tests for displaying the user's profile picture.
- [ ] Tests for displaying the user's username.
- [ ] Tests for displaying the number of posts the user has written.
- [ ] Tests for displaying a post's title.
- [ ] Tests for displaying some of the post's body.
- [ ] Tests for displaying the first comments on a post.
- [ ] Tests for displaying how many comments a post has.
- [ ] Tests for displaying how many likes a post has.
- [ ] Tests for displaying a section for pagination if there are more posts than fit on the view.
- [ ] Tests for redirecting to the post's show page when clicking on a post.

#### User Post Show Page:
- [ ] Tests for displaying the post's title.
- [ ] Tests for displaying who wrote the post.
- [ ] Tests for displaying how many comments the post has.
- [ ] Tests for displaying how many likes the post has.
- [ ] Tests for displaying the post body.
- [ ] Tests for displaying the username of each commentator.
- [ ] Tests for displaying the comment each commentator left.

If you're facing any challenges, here's a [Capybara cheat sheet](#capybara-cheat-sheet) that might help you. Feel free to reach out if you have questions or need further guidance. Keep up the good work! 🚀

Best,
Temitope

### Capybara Cheat Sheet

- [Capybara Cheat Sheet](#capybara-cheat-sheet): [Link to Capybara Cheat Sheet]

Temitope

[Link to Capybara Cheat Sheet]: <https://gist.github.com/zhengjia/428105>